src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/api/model/canonical/CanonicalAddress.kt

```diff
@@ -7,6 +7,15 @@ data class CanonicalAddress(
   val startDate: String? = "",
   val endDate: String? = "",
   val postcode: String? = "",
+  val subBuildingName: String? = "",
+  val buildingName: String? = "",
+  val buildingNumber: String? = "",
+  val thoroughfareName: String? = "",
+  val dependentLocality: String? = "",
```

Comment:
As before - this is an API. We have taken this (to me) meaningless term - see also PostTown and spread it throughout our codebase, and now we are exposing it to consumers.
If this is a MoJ wide decision to do this fair enough